src/test/scala/io/scalac/tezos/translator/Fakes.scala

```diff
@@ -0,0 +1,24 @@
+package io.scalac.tezos.translator
+
+import io.scalac.tezos.translator.routes.util.Translator
+
+object Fakes {
```

Comment:
I think it's called a mock :) Besides that, I don't see why we would put instance of Translator into an object.

Reply:
IMO Fakes and Mocks should be distinguished. I know that this is a matter of opinion, but please look f.ex. at this thread https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing

Basically `MMTranslatorFake` is a fully working lightweight implementation of `Translator` interface that is not suitable for production. 

> I don't see why we would put instance of Translator into an object.
I usually put Fakes (if there are not so many of them) to one object used in tests to clearly separate them from the rest of the codebase. Possibly other instances of "fake" characteristics could be added here in the future.
This way it is clear what is a purpose of this instances.

Reply:
The key phrase here is "if there are not so many of them" - if we keep adding them, sooner or later we'll have to split the Fakes object into separate classes, so why not do it now? We can clearly separate them from the rest of the code by putting all fakes/mocks/etc in one subpackage.

Reply:
Please be aware that you didnt give a single objective argument why to do it the way you described. 
Rephrasing your train of though - The key assumption here is " if we keep adding them", so if we can do it later, why not do it now, when it does not give us anything ?

Just not to prolong this fruitless conversation of type "my way is better" I will do it the way you want. Imo it does not change anything...